s30161-pj-NikodemCyrzan/zajecia1/instalacja-debiana-linux.md

```diff
@@ -0,0 +1,523 @@
+
+#### Accessible dark contrast installer menu
+
+Instalacja z wysokim kontrastem dla osób niedowidzących.
+
+#### Install with speech synthesis
+
+Instalacja z syntezatorem mowy dla osób niewidomych. Wymaga karty dźwiękowej.
+
+### Wybór języka
+
+> Mój wybór
+>
+> `English`, ponieważ w internecie prościej uzyskać pomoc w tym języku, niż w języku polskim.
+
+### Wybór lokalizacji
+
+> Mój wybór
+>
+> `Poland`, ponieważ umożliwia to poprawnie ustawić źródło z którego pobierane będą pakiety.
+
+### Wybór kodowania znaków
+
+> Mój wybór
+>
+> `en_US.UTF-8`, ponieważ jest to najbezpieczniejszy format kodowania znaków, który nie umożliwia wpisywania znaków diakrytycznych, a więc nie pojawią się w przyszłości z tego powodu problemy z odczytywaniem plików.
+
+### Ustawienia klawiatury
+
+> Mój wybór
+>
+> `American English`, ponieważ jest to układ większości klawiatór w Polsce.
+
+### Nadawanie nazwy hosta
+
+Jest to nazwa pod którą inne urządzenia w sieci będą widziały ten system.
+
+> Mój wybór
+>
+> `debian12`
+
+### Partycjonowanie
+
+#### Metody:
+
+-   Guided - use entire disk
+-   Guided - use entire disk and set up LVM
+-   Guided - use entire disk and set up encrypted LVM
+-   Manual
+
+> Mój wybór
+>
+> `Guided - use entire disk`, ponieważ pierwszy raz instaluję ten system i chcę przeprowadzić partycjonowanie z przewodnikiem i bez pomocy LVM.
+
+### Wybór dysku do partycjonowania
+
+Nazwa dysku:
+
+-   SCSI3 - nazwa magistrali danych którą połączony jest dysk
+-   (0, 0, 0) - kolejno:
+    -   numer kontrolera SCSI
+    -   numer identyfikacyjny dysku
+    -   numer identyfikacyjny partycji
+-   (sda) - litera "s" oznacza że jest to dysk SCSI/SATA, a "da" to kolejne litery alfabetu używane jako indeks.
+-   ciąg dalszy to nazwa dysku
+
+### Schematy partycjonowania:
+
+-   All files in one partition - wszystkie pliki przechowywane będą na jednej partycji.
+-   Separate /home partition - umieszcza dane użytkowników w osobnej partycji, która może być na innym dysku, dzięki czemu dane te mogą być bezpieczniej przechowywane.
+-   Separate /home, /var, and /tmp partitions - umieszcza /home, /var i /tmp na osobnych partycjach, dzięki czemu dane użytkowników, logi serwerów i dane tymczasowe mogą być bezpieczniej przechowywane.
+
+> Mój wybór
+>
+> `All files in one partition`, ponieważ nie umiem jeszcze używać systemów z wydzielonymi partycjami dla poszczególnych plików i nie wiem jeszcze do końca z czym to się wiąże.
+
+### Różnica między partycją primary, a logical
+
+primary - partycja przeznaczona do przechowywania systemu operacyjnego. Jest ona zapisana w konkretny sposób, dzięki czemu system może być prosto załadowany. Można na niej przechowywać również różne dane.
+
+logical - partycja przeznaczona tylko do przechowywania danych. Gdy tworzy się partycję logiczną, to tworzona jest również partycja extended (rozszerzona), która przechowywuje informacje o wszystkich partycjach logicznych na dysku.
```

Comment:
Gdy mamy włączone EFI to nie używamy partycji rozszerzonych a tym samym logicznych bo na tablicy partycji typu GPT można utworzyć maksymalnie 138 partycje podstawowe.